apps/ledger-live-mobile/src/screens/Portfolio/index.tsx

```diff
@@ -156,6 +183,18 @@ function PortfolioScreen({ navigation }: NavigationProps) {
           />
         ) : null}
       </WalletTabSafeAreaView>,
+      <Button
+        key="my-button"
+        type="shade"
+        size="large"
+        outline
+        mt={6}
+        onPress={() => {
+          Linking.openURL("ledgerlive://storyly?g=142749&instance=17650&play=sg");
+        }}
+      >
+        {`Display story with language "${trueLocale}"`}
+      </Button>,
```

Comment:
?

Reply:
This is just for testing purpose and will be removed. I needed to trigger a staging ios build with the CI